src/Shared/Constants.cs

```diff
@@ -70,14 +70,14 @@ internal static class MSBuildConstants
         internal const string MSBuildDummyGlobalPropertyHeader = "MSBuildProjectInstance";
 
         /// <summary>
-        /// The most current ToolsVersion known to this version of MSBuild as a Version object.
+        /// The most current VSGeneralAssemblyVersion known to this version of MSBuild.
         /// </summary>
-        internal static Version CurrentToolsVersionAsVersion = new Version(CurrentToolsVersion);
+        internal const string CurrentAssemblyVersion = "15.1.0.0";
 
         /// <summary>
-        /// The most current VSGeneralAssemblyVersion known to this version of MSBuild.
+        /// The current major MSBuild version for use in the automatic MSBuildAssemblyVersion property.
         /// </summary>
-        internal const string CurrentAssemblyVersion = "15.1.0.0";
+        internal const string CurrentVersionMajorMinor = "16.0";
```

Comment:
There's also [MSBuildConstants.CurrentProductVersion](https://github.com/Microsoft/msbuild/blob/ba45c1284bbc91a60692b372062ed6b03b4aec36/src/Shared/Constants.cs#L85)

Reply:
Yeah, probably better to just restore that for continuity. I'll rename and recomment to match.